### ISSUE TITLE
only first page of Jupyter Lab pdf export

### ISSUE THREAD

**Issue by SimonEnsemble** [jupyter/nbconvert#893]
Jupyter Lab 0.35.0
Ubuntu 16.04
`jupter nbconvert --version # 5.4.0`

Upon exporting a Jupyter Lab notebook to pdf, only the first page is contained in the .pdf.
Same outcome with command line:
```
jupyter nbconvert --to pdf myNB.ipynb
```

**Comment by t-makaro** [comment]
An error must be occurring when latex compiles, which stops it half way. Since latex does produce a pdf, no error is displayed.

The best way to diagnose this is to convert to latex, then compile directly with latex.

```
Jupyter nbconvert --to latex myNB.ipynb
```
which should output a file called ```myNB.tex```. Then do:
```
XeLaTeX myNB.tex
```
Which should display any compile errors.

From that we should be able to determine what is causing the conversion to stop part way.

**Comment by t-makaro** [comment]
Closing as stale / partially fixed with #896. The conversion to pdf should now produce the proper error which should let you understand why the conversion is failing. Please fell free to ask to re-open or open a new issue if you have more information and need help.
